camdecmps/procedures/refresh_emission_view_noxmasscems.sql

```diff
@@ -39,59 +45,130 @@ BEGIN
+		rpt_period_id numeric,
+		nox_hour_id character varying,
+		nox_mon_form_id character varying,
+		nox_adjusted_hrly_value numeric,
+		nox_calc_adjusted_hrly_value numeric,
+		nox_unadjusted_hrly_value numeric,
+		nox_calc_unadjusted_hrly_value numeric,
+		nox_applicable_bias_adj_factor numeric,
+		nox_calc_pct_moisture numeric,
+		nox_calc_pct_diluent numeric,
+		nox_pct_available numeric,
+		nox_segment_num numeric,
+		nox_operating_condition_cd character varying,
+		nox_fuel_cd character varying,
+		nox_modc_cd character varying,
+		h2o_hour_id character varying,
+		h2o_mon_form_id character varying,
+		h2o_adjusted_hrly_value numeric,
+		h2o_calc_adjusted_hrly_value numeric,
+		h2o_unadjusted_hrly_value numeric,
+		h2o_calc_unadjusted_hrly_value numeric,
+		h2o_applicable_bias_adj_factor numeric,
+		h2o_calc_pct_moisture numeric,
+		h2o_calc_pct_diluent numeric,
+		h2o_pct_available numeric,
+		h2o_segment_num numeric,
+		h2o_operating_condition_cd character varying,
+		h2o_fuel_cd character varying,
+		h2o_modc_cd character varying,
+		noxr_hour_id character varying,
+		noxr_mon_form_id character varying,
+		noxr_adjusted_hrly_value numeric,
+		noxr_calc_adjusted_hrly_value numeric,
+		noxr_unadjusted_hrly_value numeric,
+		noxr_calc_unadjusted_hrly_value numeric,
+		noxr_applicable_bias_adj_factor numeric,
+		noxr_calc_pct_moisture numeric,
+		noxr_calc_pct_diluent numeric,
+		noxr_pct_available numeric,
+		noxr_segment_num numeric,
+		noxr_operating_condition_cd character varying,
+		noxr_fuel_cd character varying,
+		noxr_modc_cd character varying,
+	) ON dhv.HOUR_ID = hod.HOUR_ID
+	  AND dhv.MON_LOC_ID = hod.MON_LOC_ID
+	  AND dhv.RPT_PERIOD_ID = hod.RPT_PERIOD_ID
+	JOIN camdecmps.get_monitor_hourly_values_pivoted(
+		vmonplanid, vrptperiodid, mhvParamCodes
+	) AS mhv (
+		hour_id character varying,
+		mon_loc_id character varying,
+		rpt_period_id numeric,
+		flow_hour_id character varying,
+		flow_adjusted_hrly_value numeric,
+		flow_calc_adjusted_hrly_value numeric,
+		flow_unadjusted_hrly_value numeric,
+		flow_applicable_bias_adj_factor numeric,
+		flow_pct_available numeric,
+		flow_moisture_basis character varying,
+		flow_modc_cd character varying,
+		noxc_unadjusted_hrly_value numeric,
+		noxc_applicable_bias_adj_factor numeric,
+		noxc_adjusted_hrly_value numeric,
+		noxc_calc_adjusted_hrly_value numeric,
+		noxc_modc_cd character varying,
+		noxc_pct_available numeric,
+		h2o_hour_id character varying,
+		h2o_adjusted_hrly_value numeric,
+		h2o_calc_adjusted_hrly_value numeric,
+		h2o_unadjusted_hrly_value numeric,
+		h2o_applicable_bias_adj_factor numeric,
+		h2o_pct_available numeric,
+		h2o_moisture_basis character varying,
+		h2o_modc_cd character varying,
+	) ON mhv.HOUR_ID = hod.HOUR_ID
+	  AND mhv.MON_LOC_ID = hod.MON_LOC_ID
+	  AND mhv.RPT_PERIOD_ID = hod.RPT_PERIOD_ID
+	INNER JOIN camdecmps.MONITOR_FORMULA  mf_HI 
+		ON mf_HI.MON_FORM_ID = dhv.MON_FORM_ID AND mf_HI.EQUATION_CD LIKE 'F-26%'
```

Comment:
the original code used the `DHV` alias which was the `NOX` parameter vs the `HI` parameter
```
INNER JOIN camdecmps.MONITOR_FORMULA  MF 
		ON DHV.MON_FORM_ID = MF.MON_FORM_ID AND MF.EQUATION_CD LIKE 'F-26%'
```